### PR TITLE
7.x 3.x refactor view

### PR DESCRIPTION
**Pull request by cjwest**
There are two views in this feature both with the same display URLs. One of them is disabled since it has the same machine name as in the person reference feature view. I didn't want to delete it because it is probably in use on other sites. At the same time, I don't want to continue using it since I think it could be confusing. Hence it is marked as deprecated. I'd appreciate your thoughts.

### CODE REVIEW

**Comment by cjwest** [comment]
Hey Shea, 
Please check this. I think it's ready to pull!

File: modules/stanford_course_views/stanford_course_views.info [hunk 1]
```diff
@@ -12,3 +12,5 @@ dependencies[] = views_bulk_operations
 features[ctools][] = views:views_default:3.0
 features[features_api][] = api:2
 features[views_view][] = courses
+mtime = 1424398510
+project path = sites/all/modules/stanford/stanford_courses/modules
```
Comment (sherakama) [inline modules/stanford_course_views/stanford_course_views.info]: Please remove project path.

File: modules/stanford_course_views/stanford_course_views.module [hunk 2]
```diff
@@ -5,3 +5,27 @@
  */
 
 include_once 'stanford_course_views.features.inc';
+
+function stanford_course_views_views_pre_build(&$view) {
+  // We want to provide a link to the search view when there are no results
+
+  if (($view->name == "stanford_courses_default") || ($view->name == "courses")){
```
Comment (sherakama) [inline modules/stanford_course_views/stanford_course_views.module]: stanford_courses_default should no longer exist. The two view machine names are:

courses
courses_reference

Please check that the display names are consistent between the two.

Comment (cjwest) [inline modules/stanford_course_views/stanford_course_views.module]: Good catch Shea! 

I just pushed up the change. 


Caryl 
--- 
Caryl J. Westerberg 
Web Producer, Stanford Web Services 
cjwest@stanford.edu 

swsblog.stanford.edu 

----- Original Message -----

**Comment by cjwest** [comment]
okay, removed project path.

It would be cool for me to have a check list for this stuff...

**Comment by sherakama** [comment]
The faculty courses block machine name has changed from block_1 => courses_reference_person_block

Please restore the block_1 machine name.

**Comment by cjwest** [comment]
Okay, reverted the faculty courses block.

**Comment by cjwest** [comment]
W00t! 


Caryl 
--- 
Caryl J. Westerberg 
Web Producer, Stanford Web Services 
cjwest@stanford.edu 

swsblog.stanford.edu 

----- Original Message -----

**Pull request merged**